macOS/DuckDuckGo/TabBar/View/TabBarViewItem.swift

```diff
@@ -183,6 +184,7 @@ final class TabBarItemCellView: NSView {
         closeButton.mouseOverColor = .buttonMouseOver
         closeButton.imagePosition = .imageOnly
         closeButton.imageScaling = .scaleNone
+        closeButton.sendAction(on: [.leftMouseUp, .otherMouseDown])
```

Comment:
allows middle-clicking the X to close the tab

Reply:
But the tab itself is closed by middle-down, so the suggested change just fixes the button not reacting to it the same way the tab does